packages/testing/config/jest/api/jest.setup.js

```diff
@@ -4,7 +4,6 @@
 // @NOTE without these imports in the setup file, mockCurrentUser
```

Comment:
Should this comment also be moved to around line 194 where `mockCurrentUser` is?

Reply:
Ahh I wanted to try restoring the require back to the top and testing to ensure it worked. Otherwise yeah the comment should also relocate.